courses/free-courses-en.md

```diff
@@ -805,6 +805,7 @@
 * [Essential Nextjs Typescript Tailwind Stack](https://www.youtube.com/playlist?list=PLKEkvhqFCRwL94nLP-TdIfNmbiZk5FCMx) - Fireship, Ben Awad, Traversy Media, Devs Force
 * [Flask(A Python Microframework) Tutorial](http://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world)
 * [Free Code Camp](https://www.freecodecamp.org)
+* [Full Stack Foundations](https://www.udacity.com/course/full-stack-foundations--ud088)
```

Comment:
It needs give credits to instructor and add platform either signup notes. See CONTRIBUTE to know how to,

Reply:
Hi @davorpa , made the changes.